src/src/rx_main.cpp

```diff
@@ -460,14 +460,20 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
     return true;
 }
 
-void ICACHE_RAM_ATTR HandleFreqCorr(bool value)
+uint32_t ICACHE_RAM_ATTR HandleFreqCorr(bool value)
 {
-    //DBGVLN(FreqCorrection);
+    uint32_t tempFC = FreqCorrection;
+    if (Radio.GetProcessingPacketRadio() == SX12XX_Radio_2)
+    {
+        tempFC = FreqCorrection_2;
+    }
+
+    //DBGVLN(tempFC);
```

Comment:
Any reason this should remain in the code, but be commented out? Seems like the fact that its already wrapped in the debug define is enough right?